src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
-// write your code here
+const values = [...document.querySelectorAll('span.population')].map(
+  (e) => parseInt(e.textContent.replaceAll(',', '')),
+  10,
```

Comment:
The radix parameter (10) should be passed as the second argument to the `parseInt` function. The correct usage is `parseInt(e.textContent.replaceAll(',', ''), 10)`.